data/json/obsolete.json

```diff
@@ -1124,6 +1124,54 @@
       "removal": { "skills": [ [ "mechanics", 3 ] ] }
     }
   },
+  {
+    "abstract": "door_abstract",
+    "breaks_into": "ig_vp_frame",
+    "broken_color": "cyan",
+    "broken_symbol": "&",
+    "color": "cyan",
+    "damage_modifier": 80,
+    "damage_reduction": { "all": 21 },
+    "description": "A door.  Has a window so you can see out of it, even when closed.",
+    "durability": 225,
+    "flags": [ "CARGO", "OBSTACLE", "OPENABLE", "BOARDABLE", "WINDOW" ],
+    "item": "frame",
+    "location": "center",
+    "looks_like": "t_door_metal_c",
+    "name": { "str": "door" },
+    "requirements": {
+      "install": { "skills": [ [ "mechanics", 2 ] ], "time": "60 m", "using": [ [ "vehicle_bolt", 1 ] ] },
+      "removal": { "skills": [ [ "mechanics", 2 ] ], "time": "30 m", "using": [ [ "vehicle_bolt", 1 ] ] },
+      "repair": { "skills": [ [ "mechanics", 3 ] ], "time": "60 m", "using": [ [ "welding_standard", 5 ] ] }
+    },
+    "size": 10,
+    "symbol": "+",
+    "type": "vehicle_part"
+  },
+  {
+    "copy-from": "door_abstract",
+    "id": "door_front_left",
```

Comment:
Were these supposed to be done the old way?

Reply:
Yes, because `front_left` isn't a valid variant string anymore but we need the obsolete parts for migration.